### ISSUE TITLE
Nintendo Joy Con Usage vs. Gestures

### ISSUE THREAD

**Issue by minusreality** [alvr-org/alvr-visionos#107]
Hello,

There are some community claims that a specific version of ALVR can support joy cons. Is this the case? 

When I enable gestures and launch VRChat, I cannot use my current xbox controller's thumbsticks reliably. I'm assuming because figner tracking thinks I'm doing one thing and the xbox joystick think another so they're conflicting.

When I disable gestures and launch vrchat, nothing about my controller acts as a pinch gesture or VR controller trigger click and therefore I cannot click/select anything in either SteamVR or VRChat.

What are my options here?

**Comment by shinyquagsire23** [comment]
yes, visionOS ALVR has support for Joy-Con connected to the headset, but you must turn off gestures for it to work. You can also use a DS4 or Pro Controller technically, but the bindings might be funky. Luckily the settings app allows rebinding per-app for now.

**Comment by minusreality** [comment]
It seems the support is specifically for Joy Con and not other controllers? My XBox controller has no button that will operate as a VR trigger.

**Comment by shinyquagsire23** [comment]
the controllers have to be connected to the headset, not the PC, so we're at the mercy of whatever Apple happens to support for now, unfortunately.

**Comment by minusreality** [comment]
The XBox controller is connected directly to the headset. I will purchase a Joy Con to test if it is somehow working diferrently than my current XBox controller.

**Comment by shinyquagsire23** [comment]
oh if it's actually connected and shows the rebinding stuff in settings, you should be able to use it in ALVR fine. Just change your streamer settings from Controllers -> Gestures -> Only touch to Off so that it uses the controller inputs

**Comment by austinarchibald** [comment]
Hello, trying HL Alyx for the first time with joycons, got the game loaded, but I can't move. It says move left joystick to move, it doesn't do anything. And if I hold down, it opens the menu to quit game, adjust settings, etc. Any ideas?

I'm using some joycon knock offs from Amazon, I am assuming that's not my problem. All the other buttons seem to work as expected. e.g. Triggers let me grab that antenna on the radio, turn the radio knob, etc. 

Thanks!
